# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../argus/pcap-bpf.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- argus-2.0.6.orig/common/gencode.c	2004-02-23 16:00:36.000000000 +0100
+++ argus-2.0.6/common/gencode.c	2005-09-29 11:31:59.000000000 +0200
@@ -59,7 +59,7 @@
 #include <stdlib.h>
 #include <syslog.h>
 
-#include <net/bpf.h>
+#include <pcap-bpf.h>
 
 #include <argus_out.h>
 #include <argus_filter.h>
